include/experimental/__p1673_bits/blas1_givens.hpp

```diff
@@ -372,16 +371,18 @@ void givens_rotation_setup(const complex<Real>& f,
   }
 }
 
-// c and s are std::floating_point
-template<class ElementType1,
+MDSPAN_TEMPLATE_REQUIRES(
+         class ElementType1,
          extents<>::size_type ext1,
          class Layout1,
          class Accessor1,
          class ElementType2,
          extents<>::size_type ext2,
          class Layout2,
          class Accessor2,
-         std::floating_point Real>
+         class Real,
+         /* requires */ (_MDSPAN_TRAIT(is_floating_point, Real))
```

Comment:
@youyu3 g++11 on Ubuntu 20.04 fails here with:
```shell
include/experimental/__p1673_bits/blas1_givens.hpp:384:25: error: expected identifier before ‘(’ token
  384 |          /* requires */ (_MDSPAN_TRAIT(is_floating_point, Real))
      |                         ^
```
(cases with only `Real` in `MDSPAN_TEMPLATE_REQUIRES` compile fine)

Is it something on my side ?

Reply:
Updating `mdspan` solved the problem, I'm sorry for bothering.

Reply:
Thanks for the update!  Should've noted that this needs updated `mdspan`.